scipy/special/cephes/dd_idefs.h

```diff
@@ -39,7 +41,7 @@ extern "C" {
 */
 
 /* Computes fl(a+b) and err(a+b).  Assumes |a| >= |b|. */
-static double
+static __COMP_NPY_UNUSED double
```

Comment:
Elsewhere in the codebase we've used NPY_INLINE which iiuc expands either to inline or attribute((unused))

Reply:
`numpy/core/include/numpy/npy_common.h` defines `NPY_INLINE` as one of `__inline`, `__inline__`, `inline` or empty value, but there is no mention of `(unused)` in the current numpy headers as it relates to inline.
`numpy/core/include/numpy/utils.h` defines `__COMP_NPY_UNUSED` to be one of `__attribute__ ((unused))`,  `__attribute__ ((__unused__))` or empty value.  
MSVC does have an inline directive, but doesn't appear to support unused attribute.
I see 3 options:
- `NPYLINE`
- `NPYLINE __COMP_NPY_UNUSED`
- `__COMP_NPY_UNUSED`

What is your recommendation for which to use here?

Reply:
I think in Numpy, this type of functions are just `static NPY_INLINE`